pickle.c

```diff
@@ -114,6 +114,14 @@
 #define DEBUGGING         (1)
 #endif
 
+#ifndef DEFINE_TESTS
+ #ifdef NDEBUG
+ #define DEFINE_TESTS      (0)
```

Comment:
I don't think there is any need to use **NDEBUG**, if there is a macro for **DEFINE_TESTS**.

Reply:
I agree